personalize.admin.inc

```diff
@@ -358,8 +367,12 @@ function personalize_agent_list($agents = NULL, $allow_edit = TRUE, $allow_alter
         $description = t('This table lists personalizations that use PHP (whenever possible) to display the fallback/winner variation for each variation set, which can cause variations to display more quickly than those in paused personalizations.');
         break;
     }
-    $build['status_' . $status] = array(
-      '#markup' => '<h2>' . $status_map[$status] . '</h2><p>' . $description . '</p>'
+    $build['personalization_group_' . $status] = array(
```

Comment:
And for drupallyness - '#type' => 'container', with '#attributes' => array('id' => '....').  The goal is basically not to type HTML tags.